google/ggrc-core
# GGRC-4893 Delete previous not finished import job after start new one

# Issue description

After start new import job need to remove all previous not finished jobs on BE side, because currently it is doing from the FE side

# Steps to test the changes

1. Load import file.
Result: import job entry is created.
2. Load another import file.
Expected result: the previous import job is deleted by BE.

# Solution description

If there is an active import in "Analysis" or  "In Progress" state, raise BadRequest error (401)
If there is an inactive import in "Not Started" or "Blocked" state, delete it.
Leave other imports ("Finished", "Stopped", "Analysis Failed") as they are.

# Sanity checklist




# PR Review checklist
